src/lint/mod.rs

```diff
@@ -152,7 +153,7 @@ pub mod tests {
     dml 1.4;
 
     bank sb_cr {
-        group monitor {    
```

Comment:
I don't think this requires removal. This is a test snippet and trailing spaces here are intentional